app/utils/fetchPlayers.ts

```diff
@@ -0,0 +1,19 @@
+import { getPendingPlayers } from '../hooks/server_actions';
+
+export interface PendingPlayer {
+    uuid: string;
+    username: string;
+    seatId: number;
+    buyIn: number;
+}
+
+export const fetchPendingPlayers = async () => {
```

Comment:
Is there a reason we need this wrapper aroudn the getPendingPlayers from server_actions?